staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go

```diff
@@ -68,14 +68,6 @@ type CertificateOptions struct {
 	genericclioptions.IOStreams
 }
 
-// NewCertificateOptions creates the options for certificate
-func NewCertificateOptions(ioStreams genericclioptions.IOStreams) *CertificateOptions {
```

Comment:
Rather than dropping this, add the operation to the signature. This helps consumers of this command understand all the inputs required to construct the options

Reply:
@liggitt  thanks for the review.

`NewCertificateOptions` is back